lib/content/dependabot-yml.hbs

```diff
@@ -5,7 +5,7 @@ updates:
   - package-ecosystem: npm
     directory: /
     schedule:
-      interval: daily
+      interval: {{ schedule }}
```

Comment:
this should be `interval` since `schedule` could have other attributes under it like `day` `time` etc.